clusters/k8s-infra-dev-cluster-turnup/10-clusterconfiguration.tf

```diff
@@ -120,14 +143,24 @@ resource "google_container_cluster" "cluster" {
 
   // Configure cluster addons
   addons_config {
+    horizontal_pod_autoscaling {
```

Comment:
These are the defaults. Do we need to set these?

Reply:
I called them out explicitly because I went hunting for them and didn't find them.  I thought it was useful as "documentation".  Disagree?